### PR TITLE
added docker

### PR DESCRIPTION
**Pull request by 163onmyneck**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      - MYSQL_ROOT_USERNAME=$MYSQL_ROOT_USER
+      - MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD
+      - MYSQL_DATABASE=$MYSQL_DATABASE
+    ports:
+      - "3307:3306"
+
+  app:
+    build: .
+    ports:
+      - "8081:8080"
```
Comment (okuzan) [inline docker-compose.yml]: extract ports to .env

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+version: '3.8'
+services:
+  mysql:
+    image: mysql:8.0
```
Comment (okuzan) [inline docker-compose.yml:4]: where do you set up your java app?

**Review by okuzan — CHANGES_REQUESTED** [review]
Add .env.template too

File: docker-compose.yml (hunk 2 above)
Comment (163onmyneck) [reply docker-compose.yml:4]: I set up it in Dcokerfile, no?

**Review by okuzan — APPROVED** [review]

**Pull request merged**